src/vs/vscode.d.ts

```diff
@@ -2972,6 +2972,37 @@ declare namespace vscode {
 	}
 
 	/**
+	 * An individual terminal instance within the integrated terminal.
+	 */
+	export interface Terminal {
+
+		/**
+		 * Send text to the terminal.
+		 *
+		 * @param addNewLine Whether to add a new line to the text being sent, this is normally
+		 * required to run a command in the terminal. This defaults to `true`.
+		 */
+		sendText(text: string, addNewLine?: boolean);
```

Comment:
Is this always the same sequence for new line or should `addNewLine` be something like `\n|\r\n|\r`?